pkg/k8shandler/common.go

```diff
@@ -326,6 +327,10 @@ func newEnvVars(nodeName, clusterName, instanceRam string, roleMap map[api.Elast
 			Name:  "HAS_DATA",
 			Value: strconv.FormatBool(roleMap[api.ElasticsearchRoleData]),
 		},
+		{
+			Name:  "SYSTEM_CALL_FILTER",
+			Value: strconv.FormatBool(runtime.GOARCH == "amd64"),
+		},
```

Comment:
is there a specific reason to inject this as an env var rather that just creating the configmap with this value?

Reply:
Because this is what @jcantrill suggested in https://github.com/openshift/origin-aggregated-logging/pull/1927#discussion_r443581543

Reply:
I'm a bit confused as to how this would work as a configmap, this isn't something that should be user-configurable.  Without disabling this startup check on P&Z, ES will fail.

Reply:
your change here [1] is updating the configmap already. my suggestion is to evaluate `strconv.FormatBool(runtime.GOARCH == "amd64")` there rather than letting the env var substitution occur. I don't see a need to introduce an additional variable.

[1] https://github.com/openshift/elasticsearch-operator/pull/410/files#diff-a4c0bf273d801abee649f31bda399079R7-R9